doc/user/pathd.rst

```diff
@@ -6,11 +6,168 @@ PATH
 
 :abbr:`PATH` is a daemon that handles the installation and deletion
 of Segment Routing (SR) Policies.
+Based in MPLS (This means that your OS of choice must support MPLS),
+SR add a stack of MPLS labels to ingress packets so these
+packets are egress through the desired path.
+
+.. image:: images/pathd_general.png
+
+The SR policies and Segment Lists can be configured either locally by means
+of vtysh or centralized based in a SDN controller (ODL, Cisco, ...) and
+communicated using the PCEP protocol (:rfc:`5440`).
 
 
 .. _starting-path:
 
-Starting PATH
+Configuration
+=============
+
+Explicit Segment Lists
+----------------------
+
+This is the simplest way of configuration, no remote PCE is necessary.
+In order to create a config that match the graphics used in this documentation,
+we will create a segment list (SL) called SL1 with an element for each hop and
+that element will be assigned a MPLS label.
+Then the SL1 will be used in the policy ``example1``, please note also the
+preference as in the case of multiple segment list it will be used with the
+criteria of bigger number more preference.
+Let see now the final configuration that match the graphics shown above.
+
+
+.. code-block:: frr
+
+   segment-routing
+    traffic-eng
+     segment-list SL1
+      index 10  mpls label 16001
+      index 20  mpls label 16002
+     !
+     policy color 1 endpoint 192.0.2.4
+      name example1
+      binding-sid 1111
+      candidate-path preference 100 name CP1 explicit segment-list SL1
+
+
+Explicit Segment Lists and Traffic Engineering Database (TED)
+-------------------------------------------------------------
+
+Sometimes is difficult to know the values of MPLS labels
+(adjacency changes,...).
+Based in the support of IS-IS or OSPF we can activate TED support what will
+allow pathd to resolve MPLS based in different types of segments
```

Comment:
Based on the support ...

Reply:
**Can you please share topology for the configuration for which you've configured pathd because I am unable to understand the topology with IPs itself**